### PR TITLE
Add dev.edfloreshz.Done

### PR DESCRIPTION
**Pull request by edfloreshz**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.devloop.Do.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-nightly/bin",
+        "build-args" : [
+            "--share=network"
+        ],
```
Comment (hfiguiere) [inline org.devloop.Do.json:20]: this is not allow and will fail
you need to add the cargo dependencies in the manifest.

See https://github.com/flatpak/flatpak-builder-tools

Optionaly a vendored tarball instead if git works if you use standard Rust meson templates.

File: org.devloop.Do.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "org.devloop.Do",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "master",
```
Comment (hfiguiere) [inline org.devloop.Do.json:4]: there is no `master` version. You have to use GNOME 42.

File: org.devloop.Do.json [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-nightly"
```
Comment (hfiguiere) [inline org.devloop.Do.json:7]: while this exist, I really don't recommend if you can avoid it..

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.devloop.Do.json [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            "name" : "todo",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "commit": "26f99cfb488b870fc4ddc0523e006f57f88d3796",
+                    "url" : "https://github.com/edfloreshz/do.git"
+                },
+                "build-aux/cargo-sources.json"
```
Comment (hfiguiere) [inline org.devloop.Do.json:33]: this file is missing

File: org.devloop.Do.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/rust-flatpak/cargo"
```
Comment (hfiguiere) [inline org.devloop.Do.json:19]:
```suggestion
            "CARGO_HOME" : "/run/build/todo/cargo"
```

It has to be the build dir for the module

File: org.devloop.Do.json (hunk 4 above)
Comment (edfloreshz) [reply org.devloop.Do.json:33]: Is it? I thought I included it in this [commit](https://github.com/edfloreshz/do/commit/26f99cfb488b870fc4ddc0523e006f57f88d3796)

Comment (hfiguiere) [reply org.devloop.Do.json:33]: it needs to be in this repository. It's a manifest. This is included in the manifest.

Comment (edfloreshz) [reply org.devloop.Do.json:33]: Oh, right! Sorry about that

File: org.devloop.Do.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
+{
+    "app-id" : "org.devloop.Do",
```
Comment (barthalion) [inline org.devloop.Do.json:2]: Just double-checking: do you own this domain?

**Comment by barthalion** [comment]
bot, build org.devloop.Do

File: org.devloop.Do.json (hunk 6 above)
Comment (edfloreshz) [reply org.devloop.Do.json:2]: Oh no, I forgot to change that, give me a moment.

Comment (edfloreshz) [reply org.devloop.Do.json]: Updated the domain to match one of my own, https://edfloreshz.dev is mine.

**Comment by hfiguiere** [comment]
bot, build dev.edfloreshz.ToDo

**Comment by hfiguiere** [comment]
this "3 char min" rule is ridiculous. (nothing I can do about it)

**Comment by edfloreshz** [comment]
No worries, wish it wasn't that way but what are you gonna do?

**Comment by edfloreshz** [comment]
I'll take some time to rename the app.

**Comment by edfloreshz** [comment]
App was renamed and manifest was updated.

**Comment by nedrichards** [comment]
bot, build dev.edfloreshz.Done

**Pull request closed without merging**